kalite/distributed/management/commands/retrievecontentpack.py

```diff
@@ -101,10 +134,11 @@ def extract_content_pack_metadata(zf, lang):
         shutil.copyfileobj(mf, f)
 
 
-def download_content_pack(fobj, lang):
+def download_content_pack(fobj, lang, minimal=False):
     url = CONTENT_PACK_URL_TEMPLATE.format(
         version=SHORTVERSION,
-        code=lang,
+        langcode=lang,
+        suffix="-minimal" if minimal else "",
```

Comment:
@aronasorman convention is now to append "-minimal" to content packs like you do with "en-minimal.zip", so keep doing that :)